stories/atoms/components/textField/TextField.stories.mdx

```diff
@@ -267,12 +267,23 @@ The `withBorder` property allows you to add a themed border to the textField and
 ## Multiline
 
 The `multiline` property allows you to write text on several lines. The simple text field becomes a text area.  
-When `multiline` is `true` and [disableAreaResize](/docs/atoms-textfield--disable-area-resize) is `false` the [size](/docs/atoms-textfield--sizes) is automatically `auto`.
+When `multiline` is `true` and [disableAreaResize](/docs/atoms-textfield--disable-area-resize) is `false`
+the [size](/docs/atoms-textfield--sizes) is automatically `auto`.  
+When `fullWidth` is `true`and [disableAreaResize](/docs/atoms-textfield--disable-area-resize) is `false`
+the user can only resize the height of the textField.
```

Comment:
We need to rework this whole paragraph as I can't clearly understand what it means...

If `X` is `true` and `Y` is `false`, so `Z` is ... But when `W` is `true` and `Y` is `false` etc... 

Sounds quite hard to understand easily 😅